### ISSUE TITLE
mips64el-linux fails several file tests

### ISSUE THREAD

**Issue by headius** [jnr/jnr-posix#138]
These are likely due to some file structures (stat etc) not being mapped properly.

This result was from a NeoKylin Linux on mips64el. See also jnr/jffi#75 and jnr/jnr-ffi#195.

```
Failed tests: 
  FileTest.accessTest:505 access:  expected:<-1> but was:<0>
  IOTest.testSendRecvMsg_WithControl:183 null
  ProcessTest.testGetRLimit:49 Bad soft limit for number of processes
  ProcessTest.testGetRLimitPointer:83 Bad soft limit for number of processes
  ProcessTest.testGetRLimitPreallocatedRlimit:66 Bad soft limit for number of processes
Tests in error: 
  FileStatTest.fileStatNanoTime:144 » UnsupportedOperation error: Invalid argume...
  FileStatTest.filestat:61 » UnsupportedOperation error: Invalid argument
  FileStatTest.filestatDescriptor:83 » UnsupportedOperation error: Invalid argum...
  FileStatTest.filestatDirectory:194 » UnsupportedOperation error: Invalid argum...
  FileStatTest.filestatInt:109 » UnsupportedOperation error: Invalid argument
  FileTest.fcntlDupfdTest:298 » IO Stream Closed
  FileTest.fcntlDupfdWithArgTest:320 » IO Stream Closed
  FileTest.futimeTest:178 » UnsupportedOperation error: Invalid argument
  FileTest.futimens:130 » UnsupportedOperation error: Invalid argument
  FileTest.lutimesTest:161 » UnsupportedOperation error: Invalid argument
  FileTest.utimensatAbsolutePath:115->utimensat:641 » UnsupportedOperation error...
  FileTest.utimensatRelativePath:124->utimensat:641 » UnsupportedOperation error...
  FileTest.utimesDefaultValuesTest:64 » UnsupportedOperation error: Invalid argu...
  FileTest.utimesPointerTest:93 » UnsupportedOperation error: Invalid argument
  FileTest.utimesTest:40 » UnsupportedOperation error: Invalid argument
  LinuxPOSIXTest.ioprioThreadedTest:44 » IllegalState ioprio_set is not implemen...
```

**Comment by headius** [comment]
Note that neither of those PRs have been released, so anyone working on this will need to build them and update the jnr-posix pom file to point at the snapshot builds.

**Comment by headius** [comment]
jnr/jffi#75 was merged and released as jffi 1.2.22. jnr/jnr-ffi#195 was superseded by jnr/jnr-ffi#207 which has been merged and released (shortly) as jnr-ffi 2.1.16.

jnr-posix still needs a release, but updating jnr-ffi in your Maven or Gradle build will be sufficient to pick up the newer versions.
